plugins/services/src/js/components/modals/NewCreateServiceModalForm.js

```diff
@@ -279,17 +280,25 @@ class NewCreateServiceModalForm extends Component {
     this.setState({ batch, appConfig: this.getAppConfig(batch) });
   }
 
-  handleRemoveItem(event) {
-    const { value, path } = event;
+  handleRemoveItem(event, e) {
+    const { value, path, noPropagate } = event;
     let { batch } = this.state;
 
+    if (noPropagate) {
```

Comment:
I believe the `noPropagate` condition is not needed here since the browser know how to handle the click on the specific element.